sky/serve/autoscalers.py

```diff
@@ -23,25 +24,22 @@
 class AutoscalerDecisionOperator(enum.Enum):
     SCALE_UP = 'scale_up'
     SCALE_DOWN = 'scale_down'
-    NO_OP = 'no_op'
 
 
 @dataclasses.dataclass
 class AutoscalerDecision:
     """Autoscaling decisions.
 
-    |---------------------------------------------------------|
-    | Operator   | TargetType | Meaning                       |
-    |------------|------------|-------------------------------|
-    | SCALE_UP   | int        | Number of replicas to add     |
-    |------------|------------|-------------------------------|
-    | SCALE_DOWN | List[int]  | List of replica ids to remove |
-    |------------|------------|-------------------------------|
-    | NO_OP      | None       | No scaling needed             |
-    |---------------------------------------------------------|
+    |---------------------------------------------------------------|
+    | Operator   | TargetType       | Meaning                       |
+    |------------|------------------|-------------------------------|
+    | SCALE_UP   | Dict[str, Any]   | Resource override to add      |
+    |------------|------------------|-------------------------------|
+    | SCALE_DOWN | int              | Replica id to remove          |
+    |---------------------------------------------------------------|
     """
     operator: AutoscalerDecisionOperator
-    target: Optional[Union[int, List[int]]]
+    target: Union[Optional[Dict[str, Any]], int]
```

Comment:
This is not aligned w/ docstr? i.e. docstr says it is `Dict[str, Any]`, but here type annotation is `Optional[Dict[str, Any]]`?

Reply:
good catch. let's use `Union[Dict[str, Any], int]`